sale_cutoff_time_delivery/models/sale_order.py

```diff
@@ -15,6 +15,26 @@
 class SaleOrder(models.Model):
     _inherit = "sale.order"
 
+    display_expected_date_ok = fields.Boolean(
+        string="Display Expected Date Ok", compute="_compute_display_expected_date_ok",
+    )
+
+    @api.depends(
+        "commitment_date", "expected_date",
+    )
+    def _compute_display_expected_date_ok(self):
+        """Conditions to display the expected date on the so report.
+
+        Displayed if date is set
+            - date is set
+            - policy is 'when all products are ready' and everything is available
+            - policy is 'as soon as possible' and a line is fully available
```

Comment:
```suggestion
        Displayed if:
            - date is set
            - policy is 'when all products are ready' and everything is available
            - policy is 'as soon as possible' and a line is fully available
```
Regarding the last two points the code doesn't reflect this (I suppose it will land in another glue module).

Reply:
Yes, wrong copy-paste